src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator/help-checkoutCredentialsId.html

```diff
@@ -1,3 +1,15 @@
 <div>
-    <p>Credentials used to check out sources during a build.</p>
+    <p>
+        Credentials used to check out sources during a build.
+    </p>
+    <p>
+        The default choice of "Same as scan credentials" is sufficient so long as you don't
+        need Jenkins to push some changes back into these repositories.
+    </p>
+    <p>
+        Otherwise you should create an <strong>SSH username with private key</strong> credential and
```

Comment:
To me, this [reference](https://help.github.com/articles/which-remote-url-should-i-use/) should be used to justify why we (*Jenkins developers*) recommend to use `HTTPS`.